src/analysis/multiSpecies/Microbiome_Modeling_Toolbox/simulatePairwiseInteractions.m

```diff
@@ -0,0 +1,375 @@
+function [pairwiseInteractions,pairwiseSolutions]=simulatePairwiseInteractions(pairedModels,pairedModelInfo,inputDiet,saveSolutionsFlag,numWorkers,sigD)
```

Comment:
Is this just a duplicate of the function above? 
If so reduce them t one function, if not they should have different names.